CONFIG.md

```diff
@@ -0,0 +1,151 @@
+# config
+
+The canonical documentation of shpool's config is the comments
+on the `Config` struct defined in `libshpool/src/config.rs`, but
+this document aims to provide some high level explanations of
+some common configuration options.
+
+You can specify the path to your config file by passing a
+`-c /path/to/config.toml` flag, or by creating and
+editing `~/.config/shpool/config.toml`.
+
+## Prompt Prefix
+
+By default, `shpool` will detect when you are using a shell it knows
```

Comment:
```suggestion
By default, `shpool` will detect when you are using a supported shell.
```

This would sound better I think, or I missed  the point you try to explain

Reply:
All shells are supported for use with shpool in general, it's just that this feature only works with a subset. I think saying "supported" might give the impression that shpool won't work with ksh even though it will, just without this prompt prefix feature.